ansible_runner/__init__.py

```diff
@@ -14,4 +14,3 @@
     for entry_point
     in pkg_resources.iter_entry_points('ansible_runner.plugins')
```

Comment:
As a side note, I strongly encourage migrating to importlib in place of `pkg_resources`

Reply:
I will have to do that in a separate PR.

Reply:
Sure, that's the way :)